__tests__/app.test.js

```diff
@@ -49,6 +50,58 @@ describe(" api/categories", () => {
         });
       });
   });
+  test("GET 200 - api/reviews/:review_id", () => {
+    return request(app)
+      .get("/api/reviews/2")
+      .expect(200)
+      .then((response) => {
+        expect(response.body).toMatchObject({
+          review_id: 2,
+          title: `Jenga`,
+          review_body: "Fiddly fun for all the family",
+          designer: `Leslie Scott`,
+          review_img_url:
+            "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+          votes: 5,
+          category: `dexterity`,
+          owner: `philippaclaire9`,
+          created_at: new Date(1610964101251).toString(),
+        });
+      });
+  });
+  test("GET: 200 /api/reviews/:review_id/comments", () => {
+    return request(app)
+      .get("/api/reviews/3/comments")
+      .expect(200)
+      .then((response) => {
+        expect(response.body).toEqual([
```

Comment:
Add an expect in here that checks the order of the comments coming back - most recent comments first - jest-sorted needs to be used.